bindings/node/index.d.ts

```diff
@@ -1,8 +1,19 @@
 import type { Document, Binary } from 'bson';
-import type { MongoClient, BulkWriteResult, ClientSession } from 'mongodb';
+import type { MongoClient, BulkWriteResult, ClientSession, DeleteResult, FindCursor } from 'mongodb';
 
 export type ClientEncryptionDataKeyProvider = 'aws' | 'azure' | 'gcp' | 'local' | 'kmip';
 
+export interface DataKey {
+  _id: Binary;
+  version: number;
```

Comment:
I’m not seeing `version` being mentioned anywhere – am I missing something?

Reply:
I got this schema directly from the spec - https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/client-side-encryption.rst#key-vault-collection-schema-for-data-keys

This seems to be the first time we're returning a full data key entry, so I added the schema directly.  Do you think it would be better to only return a subset of the fields?

I did notice, on re-review of the schema in the spec, that version can be unset.  So version and keyAltNames should be optional, I'll make that change.

Reply:
Yeah, I think “Implicitly 0 if unset” is the important part – I was just surprised because I’d never seen a `version` field on a data key doc :slightly_smiling_face:

Reply:
Fixed 😄